apps/website/src/pages/about/tech/commands.tsx

```diff
@@ -326,62 +330,115 @@ const AboutTechPage: NextPage = () => {
-                      link
-                    >
-                      {title}
-                      <span className="text-sm text-alveus-green-400 italic">
-                        {` (${camera.toLowerCase()})`}
-                      </span>
-                    </Heading>
-                  </dt>
-                  <dd className="mx-2">
-                    <dl className="max-w-full overflow-x-auto">
-                      {typeSafeObjectEntries(presets).map(([name, preset]) => (
+          <div className="mt-6 grid grid-cols-1 gap-6 lg:grid-cols-4">
+            {/* Camera List */}
+            <div className="col-span-1 space-y-2">
+              {/* Mobile: Dropdown */}
+              <div className="mb-2 block lg:hidden">
+                <label htmlFor="camera-select" className="sr-only">
+                  Select Camera
+                </label>
+                <select
+                  id="camera-select"
+                  value={selectedCamera}
+                  onChange={(e) =>
+                    setSelectedCamera(e.target.value as keyof typeof presets)
+                  }
+                  className="w-full rounded border border-alveus-green-200 bg-alveus-green-50 px-3 py-2 text-lg font-semibold focus:ring-2 focus:ring-alveus-green focus:outline-none"
+                >
+                  {typeSafeObjectEntries(presets).map(([camera, { title }]) => (
+                    <option key={camera} value={camera}>
+                      {title} ({camera.toLowerCase()})
+                    </option>
+                  ))}
+                </select>
+              </div>
+              {/* Desktop: Button List */}
+              <div className="hidden space-y-2 lg:block">
+                {typeSafeObjectEntries(presets).map(([camera, { title }]) => (
+                  <button
+                    key={camera}
+                    onClick={() => setSelectedCamera(camera)}
+                    className={classes(
+                      "w-full rounded px-3 py-2 text-left text-lg font-semibold",
+                      selectedCamera === camera
+                        ? "bg-alveus-green text-white"
+                        : "bg-alveus-green-50 hover:bg-alveus-green-100",
+                    )}
+                  >
+                    {title}
+                    <span className="text-sm text-alveus-green-400 italic">
+                      {` (${camera.toLowerCase()})`}
+                    </span>
+                  </button>
+                ))}
+              </div>
+            </div>
+
+            {/* Preset List */}
+            <div className="col-span-1 lg:col-span-3">
+              {selectedCamera && (
+                <Fragment key={selectedCamera}>
+                  <Heading
+                    level={3}
+                    className="scroll-mt-14 text-2xl"
+                    id={`presets:${camelToKebab(selectedCamera)}`}
+                  >
+                    {presets[selectedCamera]?.title}
```

Comment:
As an FYI, once the type changes are done that I mentioned above, `presets[selectedCamera]` should be guaranteed to be a value so you won't need optional chaining here or anywhere else